app/src/main/java/tttTDD/Java/Game.java

```diff
@@ -68,11 +76,7 @@ public boolean performMove(int playerInput) {
     }
 
     private boolean gameHasFinished() {
-        if (isThereAWinner()) {
-            gameWinner = String.format("Player %s has won", currentPlayer.getMarker());
-        } else {
-            gameWinner ="";
-        }
+        gameWinner = isThereAWinner() ? String.format("Player %s has won", currentPlayer.getMarker()) : "It's a tie";
 
         return isThereAWinner() || isThereATie();
```

Comment:
Not a new problem so not a blocker, but: what happens if `gameHasFinished` is called after `gameWinner` was set?

Reply:
It'd be a null variable no? Hard to make a case for assigning it before the game has finished.